### PR TITLE
Refactor documentation to AsciiDoc

### PR DESCRIPTION
**Pull request by yqszxx**

### CODE REVIEW

**Comment by sequencer** [comment]
Please remove makefile and use CMake instead.

**Review by sequencer — APPROVED** [review]

**Review by sinofp — CHANGES_REQUESTED** [review]
Sorry for the delay, the manual is much more readable now!
The only problem is CMakeLists.txt is not formatted, the code looks weird.

File: CMakeLists.txt [hunk 1]
```diff
@@ -58,8 +58,39 @@ set_property(TARGET espresso PROPERTY C_STANDARD 99)
...
+            COMMAND ${A2M} -o ${DST}.${SECT} ${DOCDIR}/${SRC}.adoc
+            MAIN_DEPENDENCY ${DOCDIR}/${SRC}.adoc
+    )
+
+    set(MANS ${MANS} ${DST}.${SECT})
+
+    install(
+            FILES ${CMAKE_CURRENT_BINARY_DIR}/${DST}.${SECT}
+            DESTINATION ${CMAKE_INSTALL_MANDIR}/man${SECT}
+    )
```
[the remark this answers is not in the document]
Comment (sequencer) [reply CMakeLists.txt:86]: @sinofp is it possible to add this checking in CI?

[the remark this answers is not in the document]
Comment (sinofp) [reply CMakeLists.txt:86]: > @sinofp is it possible to add this checking in CI?

Sure.

**Review by sinofp — APPROVED** [review]
🐮

**Comment by yqszxx** [comment]
Still need some rewrite on documentations.

File: doc/espresso.adoc [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+*-estrong*::
+  Uses the alternate strategy SUPER_GASP (as a replacement for LAST_
+  GASP) which is more expensive, but occasionally provides better
+  results.
+*-o[type]*::
+  Selects the output format. By default, only the ON-set (i.e., type f)
+  is output after the minimization. [type] can be one of *f*, *d*, *r*,
+  *fd*, *dr*, *fr*, or *fdr* to select any combination of the ON-set
+  (f), the OFF-set (r) or the DC-set (d). [type] may also be *eqntott*
+  to output algebraic equations acceptable to _eqntott_(1OCTTOOLS), or
```
Comment (yqszxx) [inline doc/espresso.adoc:163]: Like here, referring to something that is not provided by this repo?

Comment (yqszxx) [reply doc/espresso.adoc:163]: And `1OCTTOOLS` is not a valid man chapter.

Comment (sequencer) [reply doc/espresso.adoc:163]: You can gather all reviews, and let's schedule a meeting to discuss these.

**Comment by sequencer** [comment]
For documentation changes let's make that happen in another PR.

**Pull request merged**